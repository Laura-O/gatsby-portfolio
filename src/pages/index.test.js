import React from 'react';
import { shallow } from 'enzyme';

import IndexPage from './index';

test('rendering', () => {
  const wrapper = shallow(<IndexPage />);
  expect(wrapper).toMatchSnapshot();
});
