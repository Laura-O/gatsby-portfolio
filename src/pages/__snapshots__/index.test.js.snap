// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<div
  className="start"
>
  <StartAnimation />
  <div
    className="about-me"
  >
    I'm Laura, a full-stack developer based in Germany.
  </div>
  <Fireworks />
  <styled.div>
    <div
      className="start-button"
    >
      <GatsbyLink
        to="/about"
      >
        <styled.button>
          About
        </styled.button>
      </GatsbyLink>
    </div>
    <div
      className="start-button"
    >
      <GatsbyLink
        to="/projects"
      >
        <styled.button>
          Projects
        </styled.button>
      </GatsbyLink>
    </div>
    <div
      className="start-button"
    >
      <GatsbyLink
        to="/blog"
      >
        <styled.button>
          Blog
        </styled.button>
      </GatsbyLink>
    </div>
  </styled.div>
</div>
`;
